### PR TITLE
Update docs links in README

### LINKED ISSUE #301: Incorrect links in README.md for docs
**Description**
Incorrect links in README.md for docs
There are two such links: 
1. https://www.meilisearch.com/docs/reference/under_the_hood/typotolerance
2. https://www.meilisearch.com/docs/create/how_to/search_bar_for_docs

**Expected behavior**
Docs link mentioned in the README should point to the correct URL. 

**Current behavior**
After docs migrated, they no longer point to the correct URL

**Screenshots or Logs**
If applicable, add screenshots or logs to help explain your problem.
image

### PR DESCRIPTION
**Pull request by shivaylamba**
# Pull Request

## Related issue
Fixes #301 

## What does this PR do?
- Fixes docs links after Meilisearch docs migration

## PR checklist
Please check if your PR fulfills the following requirements:
- [x] Does this PR fix an existing issue, or have you listed the changes applied in the PR description (and why they are needed)?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ yarn playground
 Then, open your browser on the indicated URL and test the search bar 🙂
 
 The data comes from Meilisearch documentation.<br>
-Type `create an indxe` to live the Meilisearch experience with the [typo tolerance](https://docs.meilisearch.com/reference/under_the_hood/typotolerance.html).
+Type `create an index` to live the Meilisearch experience with the [typo tolerance](https://www.meilisearch.com/docs/learn/configuration/typo_tolerance).
```
Comment (bidoubiwa) [inline README.md:61]: this typo is wanted to showcase typotolerance

**Review by bidoubiwa — CHANGES_REQUESTED** [review]
Thanks for your contribution 🙏

**Comment by shivaylamba** [comment]
I have resolved the issue @bidoubiwa

**Comment by bidoubiwa** [comment]
✨ LGTM ✨

**Review by bidoubiwa — APPROVED** [review]
bors merge

**Pull request merged**